text/3759-cargo-supported-targets.md

```diff
@@ -0,0 +1,501 @@
+- Feature Name: `supported-targets`
+- Start Date: 2025-01-08
+- Pre-RFC: [Rust
+  internals](https://internals.rust-lang.org/t/pre-rfc-allow-packages-to-specify-a-set-of-supported-targets/21979)
+- RFC PR: [rust-lang/rfcs#3759](https://github.com/rust-lang/rfcs/pull/3759)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+The word _target_ is extensively used in this document. The
+[glossary](https://doc.rust-lang.org/cargo/appendix/glossary.html#target) defines its many meanings.
+Here, _target_ refers to the "Target Architecture" for which a package is built. Otherwise, the
+terms "cargo-target" and "target-triple" are used in accordance with their definitions in the
+glossary.
+
+# Summary
+
+The addition of `supported-targets` to `Cargo.toml`. This field is a `cfg` string that restricts the
+set of targets which a package supports. Packages can only be built for targets that satisfy their
+`supported-targets`.
+
+# Motivation
+
+Some packages do not support every possible `rustc` target. Currently, there is no way to formally
+specify which targets a package does, or does not support.
+
+This feature enhances developer experience when working in workspaces containing packages designed
+for many different targets. Commands run on a workspace ignore packages that don't support the
+selected target.
+
+## Long-term Motivations
+
+_These do not motivate the RFC itself, rather they motivate the [future
+possibilities](#future-possibilities) section._
```

Comment:
I disagree with this.  I'm unsure if we would approve this RFC without some of these other motivations.

```suggestion
This RFC unblocks further work to improve platform-specific packages.  While these problems are important, solving them has been left to  [future
possibilities](#future-possibilities) to deliver an MVP we can then build on.
```